test/core/integration/concrete/lockup-dynamic/streamed-amount-of/streamedAmountOf.t.sol

```diff
@@ -3,35 +3,39 @@ pragma solidity >=0.8.22 <0.9.0;
 
 import { LockupDynamic } from "src/core/types/DataTypes.sol";
 import { StreamedAmountOf_Integration_Concrete_Test } from "./../../lockup/streamed-amount-of/streamedAmountOf.t.sol";
-import { LockupDynamic_Integration_Concrete_Test } from "./../LockupDynamic.t.sol";
+import { LockupDynamic_Integration_Shared_Test, Integration_Test } from "./../LockupDynamic.t.sol";
 
 contract StreamedAmountOf_LockupDynamic_Integration_Concrete_Test is
-    LockupDynamic_Integration_Concrete_Test,
+    LockupDynamic_Integration_Shared_Test,
     StreamedAmountOf_Integration_Concrete_Test
 {
-    function setUp()
-        public
-        virtual
-        override(LockupDynamic_Integration_Concrete_Test, StreamedAmountOf_Integration_Concrete_Test)
-    {
-        LockupDynamic_Integration_Concrete_Test.setUp();
-        StreamedAmountOf_Integration_Concrete_Test.setUp();
+    function setUp() public virtual override(LockupDynamic_Integration_Shared_Test, Integration_Test) {
+        LockupDynamic_Integration_Shared_Test.setUp();
     }
 
-    function test_GivenStartTimeInPresent() external givenSTREAMINGStatus {
+    function test_GivenStartTimeInPresent() external givenSTREAMINGStatus(defaults.WARP_26_PERCENT()) {
```

Comment:
What if we inherit Modifiers from `Defaults`. can we remove the input argument from modifiers completely?

Reply:
Lets bring it back (https://github.com/sablier-labs/v2-core/pull/1058#discussion_r1819823111). I vote for modifiers without arguments. The modifier have very good names and therefore are very explicit about what they do inside. Arguments are unnecessary imo.